django_api_forms/fields.py

```diff
@@ -254,10 +263,15 @@ def to_python(self, value: str) -> typing.Optional[File]:
 
 class ImageField(FileField, IgnoreFillMixin):
     default_error_messages = {
+        'invalid_uri': _("The given URI is not a valid Data URI."),
         'invalid_image': _("Upload a valid image. The file you uploaded was either not an image or a corrupted image.")
     }
 
     def to_python(self, value) -> typing.Optional[File]:
+        if version >= "1.0.0":
```

Comment:
I think you don't have to repeat this code block because of super() call below.

Reply:
Yes, you are right. I overlooked the `super()` call. Will make new changes accordingly.